tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/RepositoryTableDefinition.java

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
That's because of `ConventionTableDefinitionsProvider` is familiar with all modules we have (have a look on method `tableDefinitionFor`).
`ConventionTableDefinitionsProvider` is used by `TableDefinitionsRepository` which uses `RepositoryTableDefinition`.
If we extract `hive` and `tpch` to external module (outside core) and keep `RepositoryTableDefinition` (and classes mentioned above) we'd end up with circular dependency (core -> hive (because of `ConventionTableDefinitionsProvider`'s `tableDefinitionFor` and hive -> core)).